safe-global/safe-react
# [Epic] Dashboard

## What it solves
Resolves #3693

Figma: https://www.figma.com/file/Y9HUUDQzrbOtMpC3Nye3lt/Safe-Dashboard?node-id=2%3A26

Adds a dashboard view for safes that includes a number of widgets:

- Overview widget
- Featured Apps widget
- Pending Transactions widget
- Safe Apps widget

## How to test

1. Open the Safe app
2. Select a safe
3. Observe being navigated to the dashboard view
4. Observe widgets arranged in a grid layout

## Screenshots
![Screenshot 2022-04-25 at 13 08 08](https://user-images.githubusercontent.com/5880855/165077705-61a8c933-e078-417d-94d1-0498e591f053.png)
